src/module/dice.js

```diff
@@ -77,59 +80,80 @@ export class OseDice {
         });
       });
     });
-  }
+  },
 
-  static digestResult(data, roll) {
-    let result = {
+  digestResult(data, roll) {
+    const result = {
       isSuccess: false,
       isFailure: false,
       target: data.roll.target,
       total: roll.total,
     };
 
-    let die = roll.terms[0].total;
-    if (data.roll.type === "result") {
-      if (roll.total === result.target) {
-        result.isSuccess = true;
-      } else {
-        result.isFailure = true;
+    const die = roll.terms[0].total;
+    switch (data.roll.type) {
+      case "result": {
+        if (roll.total === result.target) {
+          result.isSuccess = true;
+        } else {
+          result.isFailure = true;
+        }
+
+        break;
       }
-    } else if (data.roll.type === "above") {
-      // SAVING THROWS
-      if (roll.total >= result.target) {
-        result.isSuccess = true;
-      } else {
-        result.isFailure = true;
+
+      case "above": {
+        // SAVING THROWS
+        if (roll.total >= result.target) {
+          result.isSuccess = true;
+        } else {
+          result.isFailure = true;
+        }
+
+        break;
       }
-    } else if (data.roll.type === "below") {
-      // MORALE, EXPLORATION
-      if (roll.total <= result.target) {
-        result.isSuccess = true;
-      } else {
-        result.isFailure = true;
+
+      case "below": {
+        // MORALE, EXPLORATION
+        if (roll.total <= result.target) {
+          result.isSuccess = true;
+        } else {
+          result.isFailure = true;
+        }
+
+        break;
       }
-    } else if (data.roll.type === "check") {
-      // SCORE CHECKS (1s and 20s)
-      if (die === 1 || (roll.total <= result.target && die < 20)) {
-        result.isSuccess = true;
-      } else {
-        result.isFailure = true;
+
+      case "check": {
+        // SCORE CHECKS (1s and 20s)
+        if (die === 1 || (roll.total <= result.target && die < 20)) {
+          result.isSuccess = true;
+        } else {
+          result.isFailure = true;
+        }
+
+        break;
       }
-    } else if (data.roll.type === "table") {
-      // Reaction
-      let table = data.roll.table;
-      let output = Object.values(table)[0];
-      for (let i = 0; i <= roll.total; i++) {
-        if (table[i]) {
-          output = table[i];
+
+      case "table": {
+        // Reaction
+        const { table } = data.roll;
+        let output = Object.values(table)[0];
+        for (let i = 0; i <= roll.total; i++) {
```

Comment:
good observation. Internally I thought "come on now" about that one, but I understand the rationale (https://eslint.org/docs/latest/rules/no-plusplus). Since for loops aren't subject to the stated issue I suppose somewhere in the imported rule config, the allowForLoop option is set to true

Reply:
Yes, I had the same reaction when I first saw it. Makes sense with excluding loops here indeed.